spring-cloud-dataflow-shell-core/pom.xml

```diff
@@ -63,6 +63,7 @@
 		<dependency>
 			<groupId>org.json</groupId>
 			<artifactId>json</artifactId>
+			<version>20180813</version>
```

Comment:
Any specific reason for this?

Reply:
Boot 2.x removed dep handling of this.

Reply:
We probably need to start thinking to bring shell baseline fully on boot 2 as well.

Reply:
I was hoping we could avoid that....

Reply:
Eventually we can't. I was highly surprised that we got something working without re-writing shell. Hopefully we can live with this for near future but I'm sure a clean boot2 separation is needed at some point in a shell itself.